depclean-gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Do you have to keep the wrapper folder?

Reply:
Yes, it will be better if we keep the wrapper folder as it is very useful. In case if someone doesn't have Gradle installed in their system or is unupdated with the latest version then this wrapper folder will help them to build the project without any failure. For more information visit [here](https://stackoverflow.com/a/20351077/15557529)